### PR TITLE
Feature collapsible rows

### PR DESCRIPTION
**Pull request by wfortin**

### CODE REVIEW

File: scss/tables/table.scss [hunk 1]
```diff
@@ -315,4 +307,103 @@ table {
...
+    }
+
+    tbody tr td:first-child {
+      padding-left: $table-first-column-padding-left;
+    }
+  }
+
+  &.mod-collapsible-rows {
+    tr {
+      height: 45px;
```
Comment (JPLachance) [inline scss/tables/table.scss]: There's already a $table-header-height variable for the tr height. Default height and collapsible row height are not the same? If so, I would add a variable.

File: scss/tables/table.scss [hunk 2]
```diff
@@ -315,4 +307,103 @@ table {
...
+          }
+          .state-collapsed {
+            display: none;
+          }
+        }
+      }
+
+      &.heading-row {
+        &.opened td {
+          background-color: $light-grey;
```
Comment (JPLachance) [inline scss/tables/table.scss]: That color is already used for the selected state. Did you validate with the UX?

File: scss/tables/table.scss [hunk 3]
```diff
@@ -315,4 +307,103 @@ table {
...
+          padding: 0;
+
+          border-bottom: 0;
+
+          .container {
+            display: none;
+          }
+        }
+
+        &.in {
```
Comment (JPLachance) [inline scss/tables/table.scss]: &.in td instead of &.in { td {

File: scss/tables/table.scss [hunk 4]
```diff
@@ -315,4 +307,103 @@ table {
...
+          }
+        }
+
+        .columns {
+          column-count: 3;
+
+          .box {
+            page-break-inside: avoid; /* Theoretically FF 20+ */
+
+            -webkit-column-break-inside: avoid; /* Chrome, Safari */
```
Comment (JPLachance) [inline scss/tables/table.scss]: Do we need that?

**Comment by JPLachance** [comment]
The demo is still missing. Fix it then :ship: :it:! ;)

File: scss/tables/table.scss (hunk 2 above)
Comment (wfortin) [inline scss/tables/table.scss]: I'll double check in avocode.

File: scss/tables/table.scss (hunk 1 above)
Comment (wfortin) [inline scss/tables/table.scss]: I'll add a variable, because the regular height is 36px.

File: scss/tables/table.scss (hunk 2 above)
Comment (wfortin) [inline scss/tables/table.scss]: Yes this follows the mockups

File: scss/tables/table.scss (hunk 3 above)
Comment (wfortin) [inline scss/tables/table.scss]: Fixed

File: scss/tables/table.scss (hunk 4 above)
Comment (wfortin) [inline scss/tables/table.scss]: image
Yes its needed when labels or values are really long and to split the values on 3 columns

File: scss/tables/table.scss (hunk 1 above)
Comment (wfortin) [inline scss/tables/table.scss]: Done.

**Comment by wfortin** [comment]
@JPLachance I added documentation and applied changes, care to take a second look ?

Thanks.

File: docs/_includes/scripts.html [hunk 5]
```diff
@@ -83,5 +83,28 @@
...
+            $el.find('tr.collapsible-row').addClass('in');
+            $el.find('tr.collapsible-row .container').slideDown(400);
+            $el.find('[data-collapse-state]').attr('data-collapse-state', 'expanded');
+        }
+
+        function collapseRowView($el) {
+            $el.find('tr.heading-row').removeClass('opened');
+            $el.find('tr.collapsible-row').removeClass('in');
+            $el.find('tr.collapsible-row .container').slideUp(400);
+            $el.find('[data-collapse-state]').attr('data-collapse-state', 'collapsed');
```
Comment (pastjean) [inline docs/_includes/scripts.html]: pourquoi ya une partie dans la classe et une partie dans data-collapse-state

**Comment by pastjean** [comment]
C'est bon! 
Je trouve ça louche qu'on utilise des classe et des data-attributes pour un seul comportement mais sinon c'est all good

**Pull request merged**